resources/server/sv_call.ts

```diff
@@ -176,14 +178,18 @@ onNet(
       const currentCall = calls.get(transmitterNumber);
 
       const endTime = timestamp / 1000;
-
       await updateCall(currentCall, false, endTime);
-
+      const accepted = calls.get(pSource);
+      console.log(accepted)
```

Comment:
If you want to do debug logging using the logger instance
```suggestion
      callLogger.debug(`Call for ${pSource} is accepted`)
```

or something like that, otherwise no `console.log()` on the server

Reply:
I already removed the log, but can add it when the calling flow is re done.